### PR TITLE
feat(android-sdk): update native and logto ui convention

### PR DESCRIPTION
**Pull request by xiaoyijun**
## Summary

update native and logto ui convention according to this [slack disscussion](https://silverhand-io.slack.com/archives/C02A8G4HVAM/p1652930136707569).

## Linear Issue Reference

N/A

## Testing

UT

### CODE REVIEW

File: android-sdk/android/src/test/kotlin/io/logto/sdk/android/auth/logto/LogtoWebViewSocialHandlerTest.kt [hunk 1]
```diff
@@ -59,7 +59,10 @@ class LogtoWebViewSocialHandlerTest {
                 window.logtoNativeSdk = {
                     platform: 'android',
                     getPostMessage: () => (data) => window.SocialHandler.postMessage(JSON.stringify(data)),
-                    supportedSocialConnectorTargets: ['wechat-native', 'alipay-native'],
+                    supportedConnector: {
+                        universal: true,
```
Comment (darcyYe) [inline android-sdk/android/src/test/kotlin/io/logto/sdk/android/auth/logto/LogtoWebViewSocialHandlerTest.kt:63]: Per offline discussion, not sure whether we should add a comment here for further explanation on setting universal to be true.

**Review by darcyYe — APPROVED** [review]
LGTM.

File: android-sdk/android/src/main/kotlin/io/logto/sdk/android/auth/logto/LogtoWebViewSocialHandler.kt [hunk 2]
```diff
@@ -26,13 +26,16 @@ class LogtoWebViewSocialHandler(
         window.logtoNativeSdk = {
             platform: 'android',
             getPostMessage: () => (data) => window.$NAME.postMessage(JSON.stringify(data)),
-            supportedSocialConnectorTargets: [${getSupportedSocialConnectorTargets()}],
+            supportedConnector: {
+                universal: true,
```
Comment (darcyYe) [inline android-sdk/android/src/main/kotlin/io/logto/sdk/android/auth/logto/LogtoWebViewSocialHandler.kt:30]: Per offline discussion, not sure whether we should add a comment here for further explanation on setting universal to be true.

Comment (xiaoyijun) [reply android-sdk/android/src/main/kotlin/io/logto/sdk/android/auth/logto/LogtoWebViewSocialHandler.kt:31]: Added.

**Pull request merged**